docs/configexamples/l3vpn-hub-and-spoke.rst

```diff
@@ -0,0 +1,1130 @@
+requirements in order to properly implement the solution. 
+We'll use the following nodes in our LAB environment:
+
+* 2 x Route reflectors (VyOS-RRx)
+* 4 x Provider routers (VyOS-Px)
+* 3 x Provider Edge (VyOs-PEx)
+* 3 x Customer Edge (VyOS-CEx)
+
+The following software was used in the creation of this document:
+
+* Operating system: VyOS
+* Version: 1.4-rolling-202110310317
+* Image name: vyos-1.4-rolling-202110310317-amd64.iso
+
+**NOTE:** VyOS Router (tested with VyOS 1.4-rolling-202110310317) 
+–  The configurations below are specifically for VyOS 1.4.x.
+
+General information can be found in the :ref:`l3vpn-vrfs` chapter.
+
+
+
+********
+Topology
+********
+.. image:: /_static/images/L3VPN_hub_and_spoke.png
+   :width: 80%
+   :align: center
+   :alt: Network Topology Diagram
+
+
+
+*****************
+How does it work?
+*****************
+
+As we know the main assumption of L3VPN “Hub and Spoke” is, that the traffic 
+between spokes have to pass via hub, in our scenario VyOS-PE2 is the Hub PE 
+and the VyOS-CE1-HUB is the central customer office device that is responsible 
+for controlling access between all spokes and announcing its network prefixes 
+(100.100.100.100/32). VyOS-PE2 has the main VRF (its name is BLUE_HUB), its 
+own Route-Distinguisher(RD) and route-target import/export lists. 
+Multiprotocol-BGP(MP-BGP) delivers L3VPN related control-plane information to 
+the nodes across network where PEs Spokes import the route-target 60535:1030 
+(this is export route-target of vrf BLUE_HUB) and export its own route-target 
+60535:1011(this is vrf BLUE_SPOKE export route-target). Therefore, the 
+Customer edge nodes can only learn the network prefixes of the HUB site 
+[100.100.100.100/32]. For this example VyOS-CE1 has network prefixes 
+[80.80.80.80/32] / VyOS-CE2 has network prefixes [90.90.90.90/32]. 
+Route-Reflector devices VyOS-RR1 and VyOS-RR2 are used to simplify network 
+routes exchange and minimize iBGP peerings between devices.
+
+L3VPN configuration parameters table:
+
++----------+-------+------------+-----------------+-------------+-------------+
+|   Node   |  Role |     VRF    |        RD       |  RT import  |  RT export  |
++----------+-------+------------+-----------------+-------------+-------------+
+| VyOS-PE2 | Hub   | BLUE_HUB   | 10.80.80.1:1011 | 65035:1011  | 65035:1030  |
+|          |       |            |                 | 65035:1030  |             |
++----------+-------+------------+-----------------+-------------+-------------+
+| VyOS-PE1 | Spoke | BLUE_SPOKE | 10.50.50.1:1011 | 65035:1030  | 65035:1011  |
++----------+-------+------------+-----------------+-------------+-------------+
+| VyOS-PE3 | Spoke | BLUE_SPOKE | 10.60.60.1:1011 | 65035:1030  | 65035:1011  |
++----------+-------+------------+-----------------+-------------+-------------+
+
+
+
+*************
+Configuration
+*************
+
+
+
+Step-1: Configuring IGP and enabling MPLS LDP
+=====================================
```

Comment:
please fill the `=` on the same length as the headline, this will cause a warning in read the docs, due an older sphinx version

Reply:
Yes you're definitely right about Public IPs. Auto checks is a great idea to prevent this for future. I'll correct both addressing and that small issue and update this pull request, thanks!